scripting/get5_apistats.sp

```diff
@@ -123,6 +123,8 @@ static Handle CreateRequest(EHTTPMethod httpMethod, const char[] apiMethod, any:
   Handle req = SteamWorks_CreateHTTPRequest(httpMethod, formattedUrl);
   if (StrEqual(g_APIKey, "")) {
     // Not using a web interface.
+    LogError("Failed to create request because get5_web_api_key was not provided");
```

Comment:
Use the method in `http.sp` to create request handles. And move the API string check before creating the handle.

Reply:
done